dotcom-rendering/src/model/enhance-images.test.ts

```diff
@@ -3,8 +3,11 @@ import { PhotoEssay } from '../../fixtures/generated/articles/PhotoEssay';
 import { Standard as ExampleArticle } from '../../fixtures/generated/articles/Standard';
 import { images } from '../../fixtures/generated/images';
 import { blockMetaData } from '../../fixtures/manual/block-meta-data';
+import { nonEmpty } from '../web/lib/tuple';
 import { enhanceImages } from './enhance-images';
 
+if (!nonEmpty(images)) throw new Error('Empty images list');
```

Comment:
`as const` is definitely an approach we could use, but it would require more refactoring that a single line, because you then need the consumers to receive `readonly` arrays. I have given it a go and it is non-trivial, which is why I opted for a single error that should “never occur”.

Even though these arrays are fixed in our code, it’s worth noting that a change in the way the fixtures are generated could mean that in the future we have more or less items, and our tests would start failing. If this happens, the developers will not get an explicit prompt of what failed. By adding this check, we ensure that such a failure will be as explicit as possible.